tests/integration/targets/meraki_mx_content_filtering/tasks/main.yml

```diff
@@ -3,306 +3,307 @@
-    meraki_content_filtering:
-      auth_key: '{{auth_key}}'
-      org_name: '{{test_org_name}}'
-      net_name: '{{test_net_name}}'
+    cisco.meraki.meraki_mx_content_filtering:
+      auth_key: '{{ auth_key }}'
+      org_name: '{{ test_org_name }}'
+      net_name: '{{ test_net_name }}'
       state: present
       blocked_urls:
         - "http://www.ansible.com/*"
     register: single_blocked
 
-  - debug:
+  - name: Debug single_blocked
+    ansible.builtin.debug:
       var: single_blocked
 
-  - assert:
+  - name: Assert single_blocked
+    ansible.builtin.assert:
       that:
         - single_blocked.data.blocked_url_patterns | length == 1
 
   - name: Set two allowed URL pattern
-    meraki_content_filtering:
-      auth_key: '{{auth_key}}'
-      org_name: '{{test_org_name}}'
-      net_name: '{{test_net_name}}'
+    cisco.meraki.meraki_mx_content_filtering:
+      auth_key: '{{ auth_key }}'
+      org_name: '{{ test_org_name }}'
+      net_name: '{{ test_net_name }}'
       state: present
       allowed_urls:
         - "http://www.ansible.com/*"
         - "http://www.redhat.com"
     register: two_allowed
 
-  - debug:
+  - name: Debug two_allowed
+    ansible.builtin.debug:
       var: two_allowed
 
-  - assert:
+  - name: Debug two_allowed
+    ansible.builtin.assert:
       that:
         - two_allowed.changed == True
         - two_allowed.data.allowed_url_patterns | length == 2
 
   - name: Set blocked URL category
-    meraki_content_filtering:
-      auth_key: '{{auth_key}}'
-      org_name: '{{test_org_name}}'
-      net_name: '{{test_net_name}}'
+    cisco.meraki.meraki_mx_content_filtering:
+      auth_key: '{{ auth_key }}'
+      org_name: '{{ test_org_name }}'
+      net_name: '{{ test_net_name }}'
       state: present
       category_list_size: full list
       blocked_categories:
-        - "Adult and Pornography"
+        - "Adult"
     register: blocked_category
 
-  - debug:
+  - name: Debug blocked_category
+    ansible.builtin.debug:
       var: blocked_category
 
-  - assert:
+  - name: Assert blocked_category
+    ansible.builtin.assert:
       that:
         - blocked_category.changed == True
         - blocked_category.data.blocked_url_categories | length == 1
-        - blocked_category.data.url_category_list_size == "fullList"
-
-  - name: Set blocked URL category with top sites
-    meraki_content_filtering:
-      auth_key: '{{auth_key}}'
-      org_name: '{{test_org_name}}'
-      net_name: '{{test_net_name}}'
-      state: present
-      category_list_size: top sites
-      blocked_categories:
-        - "Adult and Pornography"
-    register: blocked_category
-
-  - debug:
-      var: blocked_category
-
-  - assert:
-      that:
-        - blocked_category.changed == True
-        - blocked_category.data.blocked_url_categories | length == 1
-        - blocked_category.data.url_category_list_size == "topSites"
+        # - blocked_category.data.url_category_list_size == "fullList"
```

Comment:
The value `url_category_list_size` has been missing from the output. I wrote the email to the support with that case.

Reply:
I have added the issue in the dashboard-api-python: https://github.com/meraki/dashboard-api-python/issues/198